package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kalo/ui",
-  "version": "3.8.1",
+  "version": "3.9.0",
```

Comment:
No need to make this change manually. It is done as part of the release process

Reply:
Thanks for the heads up :)
I got rid of that commit 👍